.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -132,13 +136,15 @@ jobs:
           }
       env:
         SLACK_WEBHOOK_URL: ${{ secrets.SLACK_LEVITATE_WEBHOOK_URL }}
+        HAS_SECRETS: ${{ (github.repository == "grafana/grafana" || secrets.SLACK_LEVITATE_WEBHOOK_URL != '') || '' }}
```

Comment:
this one should stay i guess

Reply:
FYI/TIL - Apparently strings must be in github actions expressions, strings must be quoted with _single_  quotes.

https://docs.github.com/en/actions/learn-github-actions/expressions#literals 

![image](https://github.com/grafana/grafana/assets/46142/eacd4f00-f0e5-4114-85b1-f2e655708bc8)

 I will fix this and push up.